src/components/Aside/Aside.jsx

```diff
@@ -1,138 +1,313 @@
-import React from "react";
+import React, { useState } from "react";
+import { useFilter } from "../../context/filter-context";
 import "./Aside.css";
+
 export function Aside() {
+  const {
+    filterDispatch,
+    filter: {
+      sortBy,
+      rating,
+      sliderValue,
+      categoryAction,
+      categorySports,
+      categoryRPG,
+    },
+  } = useFilter();
+  const [filterMobile, setFilterMobile] = useState(false);
+  const [sortMobile, setSortMobile] = useState(false);
   return (
     <>
       <aside>
         <span className="filter-title">
           <h2>Filters</h2>
-          <button className="btn btn-text-secondary"> clear</button>
+          <button
+            onClick={() => filterDispatch({ type: "RESET" })}
+            className="btn btn-text-secondary"
+          >
+            clear
+          </button>
         </span>
         <h2>Price</h2>
-        <input type="range" className="slider" />
+        <div className="slider-container">
+          <p className="slider_value">Rs.0</p>
+          <input
+            type="range"
+            min={500}
+            max={3500}
+            step={100}
+            value={sliderValue}
+            onChange={(e) =>
+              filterDispatch({
+                type: "SLIDER",
+                payload: Number(e.target.value),
+              })
+            }
+            className="slider"
+          />
+          <p className="slider_value">Rs.{sliderValue}</p>
+        </div>
         <h2>Category</h2>
         <span>
-          <input type="checkbox" />
+          <input
+            type="checkbox"
+            checked={categoryAction}
+            onChange={() => filterDispatch({ type: "ACTION" })}
+          />
           <label htmlFor="Action">Action</label>
```

Comment:
- you have included for attribute for all label tags, also add id to the all of the inputs for better accessibility.
Rest everything looks great, Nice work akshay.